.travis.yml

```diff
@@ -1,54 +1,66 @@
+    # gh (GbGD)
+    - secure: GbGD/yL0mIrM+66ACCR0v/tBe0Z2Qy5tkJ+Z8PYOLfFwE9ztkkH6Wfx93RTnolom8GQAnQQQ76zlnKYbW1PzGcGqCZ3Rmbk/0TxyAh2vJ0YRlF2xVWG2txr9p0am8nuS9po4M4dQ0wDBsh2drqeidlR6xe0ZiDh2/4bqmRSuQPt4Gf7a0rDIFSaqvG/dLUn3X9+AnZqpLuo9Y8ZoxL4XuPTTpQ1vcCHcOzGQLLCCet9jMxmnTRnrsOl1g7ypL+5PY7VgeTIYuWlKI5rg4ntcrpkRZ2CtpU2UeLU01AnwgbbU5y7DokL05P0Ixx4eGLXrVWeAi6lSHZLCNe3N9WXJGcsmHnNa8OhwjkX+5lLpj9vyVt7FNC/MEIEpzhM81Wu1W8ABcxfa2cSCS4cetWlGCfviwqkzCX2/Tsz3xFFElZJEYTX3ez9unvriU0E8jkw/ZAhnVxHbMpB5ekZbyjRK6sOFLvyMz+IK5j8gfYsp+HfiTCdeDxs/ExAh5cTBJSPebcBZCj+jgUb7Hn/6GKwoeBcApHnAew15bzM+zH8uK6pk2X31vhfrCbOFgdGG/7SBzlJKSdk7mUf2EIswbq1zDSCIeDMXWtjjd+xevXNp3wXxMvIbZEzyzxda8YxpqRgSrP1l1pi1pOhRFN/AkXP8KjXQ5KMGECxPzC1s4sf1sYs=
 
 jobs:
   fail_fast: true
   allow_failures:
     - env: EMBER_TRY_SCENARIO=typescript-beta
     - env: EMBER_TRY_SCENARIO=ember-cli-beta
-
   include:
-    - stage: "Tests"
-      name: "Tests"
+    # Scenario: obey lockfile
+    - stage: 'Tests @ Locked Dependencies'
+      name: 'Tests @ Locked Dependencies'
+      install:
+        - yarn install --non-interactive # omitted usual --no-lockfile
       script:
-        - test $MD_ONLY && echo "Skipped!" || yarn lint:js
-        - test $MD_ONLY && echo "Skipped!" || yarn ci:prepare
-        - test $MD_ONLY && echo "Skipped!" || yarn ci:test
-    - stage: "Versioned Tests"
+        - test $MD_ONLY && echo "Skipped! 'yarn lint:js'"    || yarn lint:js
+        - test $MD_ONLY && echo "Skipped! 'yarn ci:prepare'" || yarn ci:prepare
+        - test $MD_ONLY && echo "Skipped! 'yarn ci:test'"    || yarn ci:test
+
+    # Scenario: lockfile deleted & re-created
+    - name: 'Floating Dependencies'
+      script:
+        - test $MD_ONLY && echo "Skipped! 'yarn ci:prepare'" || yarn ci:prepare
+        - test $MD_ONLY && echo "Skipped! 'yarn ci:test'"    || yarn ci:test
+
+    - stage: 'Supported TypeScript & Ember-CLI Versions'
       env: EMBER_TRY_SCENARIO=typescript-release
     - env: EMBER_TRY_SCENARIO=typescript-beta
     - env: EMBER_TRY_SCENARIO=ember-cli-release
+      node_js: 10 # slightly more modern node, so travis-deploy-once selects this job for publish
+      script:
+        - test $MD_ONLY && echo "Skipped!" || yarn ci:prepare
+        - test $MD_ONLY && echo "Skipped!" || yarn ember try:one $EMBER_TRY_SCENARIO && yarn travis-deploy-once "yarn semantic-release"
```

Comment:
Because this test is run against a slightly more modern node version, it will always be selected by https://github.com/semantic-release/travis-deploy-once as the one that does the publish. Because of this, it's also the only one that bothers to cleanup after `ember-try` stuff is done.